.github/vale/dicts/aiven.dic

```diff
@@ -85,6 +85,7 @@ hyperloglogs
 hypertable
 hypertables
 InfluxDB
+influx
```

Comment:
why is this needed?

Reply:
> why is this needed?

Vale issue. ``influx -import`` was not a recognized word.

Reply:
ok

Reply:
I believe the `influx -import` should be seen as code, therefore the line shouldn't be needed

Reply:
double-checked, it is in code style, and yet I received the vale error. Hence I added it previously. Maybe I can remove it and see what happens.